airflow-core/src/airflow/api_fastapi/core_api/datamodels/ui/config.py

```diff
@@ -24,13 +24,13 @@
 class ConfigResponse(BaseModel):
     """configuration serializer."""
 
-    page_size: int
-    auto_refresh_interval: int
-    hide_paused_dags_by_default: bool
+    page_size: int | None
+    auto_refresh_interval: int | None
+    hide_paused_dags_by_default: bool | None
     instance_name: str
     enable_swagger_ui: bool
```

Comment:
What about other attributes? Can they not be None?

Reply:
don't know; but, webserver does not crash now.